IndyAV/simulation/indyav_gazebo/xacro/go_kart.xacro

```diff
@@ -8,18 +8,18 @@
   <xacro:property name="chassis_length" value = "1.0"/> <!-- x -->
   <xacro:property name="chassis_width" value = "0.75"/>  <!-- y -->
   <xacro:property name="chassis_height" value = "0.25"/> <!-- z -->
-  <xacro:property name="chassis_mass" value = "50.0"/>
+  <xacro:property name="chassis_mass" value = "50.00"/>
 
   <xacro:macro name="body_geometry">
     <geometry>
-      <box size="${chassis_length} ${chassis_width} ${chassis_height}" />
+      <box size="${chassis_width} ${chassis_length} ${chassis_height}" />
     </geometry>
   </xacro:macro>
 
   <xacro:macro name="body_inertial">
     <inertial>
       <mass value="${chassis_mass}"/>
-      <xacro:box_inertia x="${chassis_length}" y="${chassis_width}" z="${chassis_height}" mass="${chassis_mass}"/>
+      <xacro:box_inertia x="${chassis_width}" y="${chassis_length}" z="${chassis_height}" mass="${chassis_mass}"/>
```

Comment:
why are these swapped? generallly with the base link, the x should be forward(length), the y to the left (width), and the z up and down (height) but you have those switched here.

ie if you take your right hand and make a finger gun ad point it in front of you, then extend you middle finger out to the left, you end up with the xyz system (index finger, middle finger, and thumb)